### PR TITLE
chore: add replica field to be configurable

### PR DESCRIPTION
**Pull request by hectorj2f**
Signed-off-by: hectorj2f <hectorf@vmware.com>

 **Description of the change**

The replica field wasn't configurable via values.yaml

I used `helm schema-gen` plugin to generate the schema.

 **Existing or Associated Issue(s)**

 **Additional Information**

 

**Checklist**

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). Where applicable, update and bump the versions in any associated umbrella chart
- [x] Variables are documented in the `values.yaml` and added to the README.md. THe [helm-docs](https://github.com/norwoodj/helm-docs) utility can be used to generate the necessary content. Use `helm-docs --dry-run` to preview the content
- [x] JSON Schema generated
- [x] List tests pass for Chart using the [Chart Testing](https://github.com/helm/chart-testing) tool and the `ct lint` command

### CODE REVIEW

**Comment by hectorj2f** [comment]
This chart version needs an update

File: charts/policy-controller/README.md [hunk 1]
```diff
@@ -20,7 +20,9 @@ The following table lists the configurable parameters of the policy-controller c
 | policywebhook.extraArgs | object | `{}` |  |
 | policywebhook.image.pullPolicy | string | `"IfNotPresent"` |  |
 | policywebhook.image.repository | string | `"gcr.io/projectsigstore/policy-webhook"` |  |
-| policywebhook.image.version | string | `"sha256:5ddc8c4c7f609a6a8396f469fde685e8774f7ee3aa4ef56f18adcb90b546a48c"` |  |
+| policywebhook.image.version | string | 
+`"sha256:5ddc8c4c7f609a6a8396f469fde685e8774f7ee3aa4ef56f18adcb90b546a48c"` |  |
```
Comment (k4leung4) [inline charts/policy-controller/README.md:24]: should be all on one line.

Comment (hectorj2f) [reply charts/policy-controller/README.md:24]: it doesn't seem to be a problem for the rendering.

**Comment by hectorj2f** [comment]
Rebased :)

File: charts/policy-controller/values.schema.json [hunk 2]
```diff
@@ -1,1281 +1,285 @@
 {
-    "$schema": "http://json-schema.org/draft-07/schema",
-    "$id": "http://example.com/example.json",
+    "$schema": "http://json-schema.org/schema#",
```
Comment (cpanato) [inline charts/policy-controller/values.schema.json:2]: all those remove are correct?

Comment (hectorj2f) [reply charts/policy-controller/values.schema.json:2]: i used the `helm schema-gen` plugin, as discussed in #https://github.com/sigstore/helm-charts/issues/192. I assumed it is a valid operation.

**Review by cpanato — DISMISSED** [review]

**Review by k4leung4 — APPROVED** [review]

**Pull request merged**